codebase/config/sync/views.view.study_administration.yml

```diff
@@ -1,6 +1,6 @@
 uuid: 0b3fd574-ca20-42bf-80ba-36c4d92d73df
 langcode: en
-status: true
+status: false
```

Comment:
what does this do?

Reply:
This change will disable the view and keep it in the disabled views section.